sys/net/gnrc/link_layer/lorawan/gnrc_lorawan.c

```diff
@@ -58,6 +58,7 @@ static inline void gnrc_lorawan_mcps_reset(gnrc_lorawan_t *mac)
     mac->mcps.waiting_for_ack = false;
     mac->mcps.fcnt = 0;
     mac->mcps.fcnt_down = 0;
+    gnrc_lorawan_set_uncnf_redundancy(mac, CONFIG_LORAMAC_DEFAULT_REDUNDANCY);
```

Comment:
Shouldn't this be in MLME ?

Reply:
I think there is no clear yes or no here. So let's go with the current approach.

Reply:
It is in fact clear, because this goes below the MLME interface and closer to the "submac" part of LoRaWAN. Therefore, it should stay as it is.